### PR TITLE
Codebase Restructure

### PR DESCRIPTION
**Pull request by kartikcho**
Moved code out of App.js into separate files and folders for better organization and structure.

### TODO

- [x] Run prettier

### CODE REVIEW

File: src/canvas/renderCanvas.js
Comment (taneliang) [inline src/canvas/renderCanvas.js]: This TODO for `renderCanvas` should still be put with `renderCanvas`

File: src/tracerbench/trace/index.js
Comment (taneliang) [inline src/tracerbench/trace/index.js]: `getBrowserArgs` doesn't exist

File: src/App.js
Comment (taneliang) [inline src/App.js]: A lot of these constants aren't being used. It may be a good idea to not import these. I think we can leave them in constants.js and clean them up in the future.

**Review by taneliang — COMMENTED** [review]
Quite a few import paths are broken; you may want to check those again.

I think we should also rename util/utils.js to util/getBatchRange.js since we now have a util folder.

This PR is too big to review so I haven't looked at the code in detail, but the app still seems to work.

But apart from that (and the comments below) this is a good big first step i the right direction. I think the organization is still not ideal but we can fix that in the future when this feels messy again.

App.js will likely need to be split up further as our profiler will have more than one page, but we can do that in another PR so that we can land this quickly.

Requesting review from @jevakallio to see if he has any tips for us :)

**Comment by kartikcho** [comment]
Most of the broken imports were caused by vscode import updates (which is why the earlier commit was failing), apparently `speedscope` had it's own `utils.js` and tracerbench has references to utils too (which doesn't exist as you mentioned in the comment above). 
I've fixed almost all of them and renamed `utils.js` but let me know if you find some left out.

File: src/canvas/renderCanvas.js
Comment (taneliang) [inline src/canvas/renderCanvas.js]: These imports should be at the top of the file. We should add a linter asap to catch these things automatically (I've opened #16)

**Review by taneliang — APPROVED** [review]
> apparently speedscope had it's own utils.js and tracerbench has references to utils too (which doesn't exist as you mentioned in the comment above).

Uhoh. Let's hope we can just black box speedscope and tracerbench (since they're third party code) and hope those missing imports don't break anything.

Approving, but let's wait for @jevakallio's review before merging.

File: src/canvas/renderCanvas.js
Comment (kartikcho) [reply src/canvas/renderCanvas.js]: Agreed, we can update issues like these in a PR that adds linter and fixes said linter errors. For now, let's leave it in.

Comment (taneliang) [reply src/canvas/renderCanvas.js]: I think we should fix this here, because a random import in the middle of a file is very surprising. We can fix less obvious issues later with the linter's help but since this is new code we should try to keep a certain standard haha.

Comment (kartikcho) [reply src/canvas/renderCanvas.js]: Alright, done! (Also did an in browser commit, don't @ me)

File: src/canvas/canvasUtils.js
Comment (jevakallio) [inline src/canvas/canvasUtils.js:1]: I recommend using `git mv` to move a file  and commit it, since now I can't see if the file has been changed or just moved. This way we also lose the version history.

No need to change anything now, only for future reference.

**Review by jevakallio — APPROVED** [review]
LGTM! Did not review renderCanvas and canvasUtils in detail.

File: src/canvas/canvasUtils.js
Comment (kartikcho) [reply src/canvas/canvasUtils.js:1]: I created this file in this PR but I'll keep that in mind, thanks!

**Pull request merged**
